<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>소프트웨어융합대학 사물함 신청 시스템</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM Hanna 11yrs Old</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LockerSpot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ON국민 아이디:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>비밀번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>관리자 로그인</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
